_sass/layout/header.scss

```diff
@@ -44,20 +44,20 @@
     &.header-home {
         background: none;
         .header-background {
-            background: rgba($gray-darker, 0.45);
+            background: linear-gradient($brand-secondary, 40%, $gray-darker);
             position: relative;
-            padding-bottom: 150px;
+            padding-bottom: 120px;
 
-            &:before {
-                content: "";
-                position: absolute;
-                background: url("../img/frontpage/background-header-home.jpg") no-repeat center center;
-                @include image-size();
-                left: 0;
-                width: 100%;
-                height: 100%;
-                z-index: -1;
-            }
+            // &:before {
+            //     content: "";
+            //     position: absolute;
+            //     background: url("../img/frontpage/background-header-home.jpg") no-repeat center center;
+            //     @include image-size();
+            //     left: 0;
+            //     width: 100%;
+            //     height: 100%;
+            //     z-index: -1;
+            // }
```

Comment:
This should probably be removed instead of commented out.

Reply:
gone